src/upload-tests.proj

```diff
@@ -102,8 +102,19 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="CopyTestILCToTestHostDir" Condition="'$(BuildingUAPAOTVertical)' == 'true'">
+    <PropertyGroup>
+      <_TestILCFolderCopy>$(TestSharedFXDir)TestILC/</_TestILCFolderCopy>
```

Comment:
As discussed before, I strongly favor this being a separate folder which we include as a separate correlation payload.